res/unix/Makefile

```
@@ -1,22 +1,47 @@
-POLARIS_BIN_DIR := ~/.local/bin/polaris
-POLARIS_DATA_DIR := ~/.local/share/polaris
+.PHONY: all build install install-bin install-data clean uninstall
+
+PREFIX ?= /usr/local
+
+EXEC_PREFIX ?= $(PREFIX)
+BINDIR ?= $(EXEC_PREFIX)/bin
+
+DATAROOTDIR ?= $(PREFIX)/share
+DATADIR ?= $(DATAROOTDIR)
+LOCALSTATEDIR ?= $(PREFIX)/var
+RUNSTATEDIR ?= $(LOCALSTATEDIR)/run
+
+POLARIS_BIN_PATH := $(BINDIR)/polaris
+export POLARIS_WEB_DIR :=  $(DATADIR)/polaris/web
+export POLARIS_SWAGGER_DIR :=  $(DATADIR)/polaris/swagger
+export POLARIS_DB_DIR := $(LOCALSTATEDIR)/lib/polaris
+export POLARIS_LOG_DIR := $(LOCALSTATEDIR)/log/polaris
+export POLARIS_CACHE_DIR := $(LOCALSTATEDIR)/cache/polaris
+export POLARIS_PID_DIR := $(RUNSTATEDIR)/polaris
 
 all: build
 
 build:
 	cargo build --release
 
-install: build
-	install -d $(POLARIS_BIN_DIR)
-	install -d $(POLARIS_DATA_DIR)
-	install ./target/release/polaris $(POLARIS_BIN_DIR)
-	cp -r ./web $(POLARIS_DATA_DIR)
-	cp -r ./swagger $(POLARIS_DATA_DIR)
-	@echo "Polaris installation complete!"
+install: install-bin install-data
+
+install-bin: build
+	install -Dm755 ./target/release/polaris $(POLARIS_BIN_PATH)
+
+install-data:
+	install -d $(POLARIS_WEB_DIR)
+	install -d $(POLARIS_SWAGGER_DIR)
+	cp -r ./web $(POLARIS_WEB_DIR)
+	cp -r ./swagger $(POLARIS_SWAGGER_DIR)
```

This makes a redundant web/web directory. The `-T` flag fixes this:
```suggestion
	cp -rT ./web $(POLARIS_WEB_DIR)
	cp -rT ./swagger $(POLARIS_SWAGGER_DIR)
```